package.json

```diff
@@ -54,7 +46,8 @@
     "graphql": "^15.5.2",
     "graphql-tag-ts": "^0.1.16",
     "lodash": "^4.17.21",
-    "nodemailer": "^6.6.3",
+    "material-ui-dropzone": "^3.5.0",
```

Comment:
on the other hand.. this doesn't seem to be used

Reply:
deleted